xml/System.Net.Security/SslClientAuthenticationOptions.xml

```diff
@@ -128,8 +128,10 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a value that indicates whether the SslStream should allow TLS resumption.</summary>
+        <value>
+        <see langword="true" /> to indicate that the <see cref="T:System.Net.Security.SslStream" /> allows TLS resumption; otherwise, <see langword="false" />. The default value is <see langword="true" />.
```

Comment:
```suggestion
          <see langword="true" /> if the <see cref="T:System.Net.Security.SslStream" /> allows TLS resumption; otherwise, <see langword="false" />. The default value is <see langword="true" />.
```